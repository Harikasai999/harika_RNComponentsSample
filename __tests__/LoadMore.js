import 'react-native'
import React from 'react'
import LoadMore from '../Src/LoadMore'
import renderer from 'react-test-renderer'
import 'isomorphic-fetch'
it('renders correctly', () => {
  const loadmore = renderer.create(<LoadMore />).toJSON()
  expect(loadmore).toMatchSnapshot()
})
