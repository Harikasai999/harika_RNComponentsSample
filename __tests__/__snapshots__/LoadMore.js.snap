// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#ecf0f1",
      "flex": 1,
      "justifyContent": "center",
      "paddingTop": 20,
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#999999"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
